src/lib/testing/cypress/mock/mockutility.ts

```diff
@@ -0,0 +1,294 @@
+/// <reference types="cypress" />
+
+declare global {
+  // tslint:disable-next-line
+  namespace Cypress {
+    // tslint:disable-next-line
+    interface Chainable {
+      now(func: string, ...args: any[]): Chainable<Element>;
+    }
+  }
+}
+
+import { login, logout, ILoginCredentials, waitForAngular } from '../utility/utility';
+
+import * as moment_ from 'moment';
+const moment: any = moment_;
```

Comment:
Should be typed

Reply:
Fixed. Should have referenced adding scripts to any projects using this utility.